js/index.js

```diff
@@ -20,7 +20,7 @@ INSTRUCTIONS:
     project, when tests are failing, 
     you should get helpful errors 
     along the way!
-    ************/
```

Comment:
Add a professional read me file, use the template from [https://github.com/microverseinc/readme-template/blob/master/README.md](url)

Reply:
- [ ] Kindly add a meaningful code and review that. You mentioned the README file but have not seen the README file; this project aims to learn how to do a review and learn HTML and CSS codes. In giving a code review, please be polite and clear about the inline comment you give feedback on, and explain why it required changing aspects.

Reply:
What do you mean I have not seen the README file? its there, it just has nothing in it, so I made a request to add a proper readme file.